messenger.rst

```diff
@@ -501,112 +467,84 @@ you can disable them like this:
         # config/packages/messenger.yaml
         framework:
             messenger:
-                buses:
-                    messenger.bus.default:
-                        default_middleware: false
+                transports:
+                    async_priority_high:
+                        dsn: '%env(MESSENGER_TRANSPORT_DSN)%'
+                        options:
+                            # queue_name is specific to the doctrine transport
```

Comment:
I don't understand this -> `queue_name is specific to the doctrine transport`  Do you mean that the allowed values of `queue_name` are restricted and depend on the queue type?